runtime/sync.c

```diff
@@ -29,6 +29,11 @@
 /* System-dependent part */
 #include "sync_posix.h"
 
+CAMLexport void caml_check_error(int retcode, char * msg)
+{
+  return sync_check_error(retcode, msg);
+}
```

Comment:
This should be (even though the function is rewritten in later commits)
```c
CAMLexport void caml_check_error(int retcode, char * msg)
{
  sync_check_error(retcode, msg);
}
```

to prevent this MSVC warning:
```
runtime/sync.c(38): warning C4098: 'caml_check_error': 'void' function returning a value
```